src/transaction.ts

```diff
@@ -2278,6 +2279,20 @@ export class Transaction extends Dml {
   useInRunner(): void {
     this._useInRunner = true;
   }
+
+  /**
+   * Use optimistic concurrency control for the transaction.
```

Comment:
Should we mention in comments that its a restricted feature ?

Reply:
The feature is restricted on the server side. It means that when the feature is GA, users can still use the current library version to try OCC(including DML).